packages/bbui/src/Drawer/Drawer.svelte

```diff
@@ -6,8 +6,7 @@
 
   export let title
   export let fillWidth
-
-  let visible = false
+  export let visible = false
```

Comment:
Not sure about exposing this as a prop, because it allows control by both this prop and also the existing show/hide API that we export.